api/buf.gen.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
```

Comment:
I've never used buf, but it seems fancy.

Reply:
Yeah, the additional configuration would bloat the protoc command and plugins are somewhat nicer with buf